### PR TITLE
Update list of contributors 

### PR DESCRIPTION
**Pull request by abbra**
Another part of the preparation for the 4.7 release: update list of contributors.

I tried to sort the list of translators, thus there is a bit more churn in that part.

### CODE REVIEW

**Comment by abbra** [comment]
Note that pagure's hook to push to github seems to be broken again, so you can see more commits in this PR than it actually has.

File: Contributors.txt [hunk 1]
```diff
@@ -108,25 +109,29 @@ Developers:
...
 	Lenka Ryznarova
+	Alexander Scheel
 	Thorsten Scherf
 	shanyin
 	Kaleemullah Siddiqui
 	Michael Simacek
 	Lars Sjostrom
 	Filip Skola
 	Aleksei Slaikovskii
+	Alexey Slaykovsky
```
Comment (rcritten) [inline Contributors.txt]: Should we ask him which name he prefers?

Comment (abbra) [reply Contributors.txt]: @slaykovsky which name do you prefer to use in the Contributors list?

Comment (abbra) [reply Contributors.txt]: Ok, let me just drop the new addition for @slaykovsky  and keep using the old one right now.

**Comment by flo-renaud** [comment]
master:

* 753264069f29e47bf222e50e95a7ec5849a7f6cb Update list of contributors

**Pull request closed without merging**